### ISSUE TITLE
Picture of text not accessible for screen readers. 

### ISSUE THREAD

**Issue by stolenletters** [FreeCodeCamp/FreeCodeCamp#10968]
#### Browser Information

* Browser Name, Version: 
* Operating System: 
* Mobile, Desktop, or Tablet: 

#### Your Code

```js
 
 
 
```

**Comment by systimotic** [comment]
Hi! I'm new here :wave: :grin:  
I looked into this issue a little bit. The challenge we're talking about is [Use the Bootstrap Grid to Put Elements Side By Side](https://www.freecodecamp.com/challenges/use-the-bootstrap-grid-to-put-elements-side-by-side). It includes an image in the description, which can be found here https://i.imgur.com/FaYuui8.png This is also where you go if you click on the image. I think the source of this is in the [Bootstrap documentation over here](https://v4-alpha.getbootstrap.com/layout/grid/#example-stacked-to-horizontal). The best solution I can come up with to solve this is linking the image to the source in the documentation. I realize that there is still only a visual representation there, so it's still not perfect. An different option that could be worth a try is setting the alt or title to a detailed explanation. It would be great if we could get some input from other people on what they think is the best solution. I'd love to try to implement it once that's decided.

**Comment by dhcodes** [comment]
I had the same though. Link it to the docs.
